inji-web/src/components/DataShare/DataShareContent.tsx

```diff
@@ -0,0 +1,99 @@
+                </label>
+
+                <label className="w-1/2 flex items-center space-x-2" data-testid={"DataShareContent-Validity-Date"}>
+                    <input
+                        type="radio"
+                        name="consentValidity"
+                        value="date"
+                        disabled={true}
+                        className="accent-iw-primary scale-150"
+                    />
+                    <span className="font-light text-iw-subTitle" data-testid={"DataShareContent-Validity-Date-Title"}>{t("content.validityDate")}</span>
+                </label>
+            </div>
+        </div>
+        <div className="relative flex mb-4" onClick={()=>setTimesDropDown(times => !times)}>
+            <div className={"w-1/3"}></div>
+            <div className={"w-2/3 py-4 px-4 rounded-lg border-2 border-iw-borderLight flex flex-row items-center"}>
+                <label className={"w-full h-full"} data-testId={"DataShareContent-Selected-Validity-Times"}>{getExpiryDisplayName(vcExpiryTimes)}</label>
+                <MdOutlineKeyboardArrowDown size={30} color={'var(--iw-color-arrowDown)'} />
+            </div>
+        </div>
+
+        {timesDropDown && <div className="relative flex mb-4">
+            <div className={"w-1/3"}></div>
+            <div
+                className={"w-2/3 py-4 px-2 border-2 border-iw-borderLight rounded-lg shadow-lg shadow-iw-shadow flex flex-col justify-center items-center"} data-testid={"DataShareContent-Validity-Times-DropDown"}>
+                <label data-testid={"DataShareContent-Validity-Times-DropDown-Once"} onClick={()=>{dispatch(storeVCExpiryTimes(1)); setTimesDropDown(false)} } className={"w-full h-full py-3 px-4 hover:bg-iw-borderLight hover:rounded-lg"}>{t("content.validityTimesOptions.once")}</label>
+                <label data-testid={"DataShareContent-Validity-Times-DropDown-Thrice"} onClick={()=>{dispatch(storeVCExpiryTimes(3)); setTimesDropDown(false)} } className={"w-full h-full py-3 px-4 hover:bg-iw-borderLight hover:rounded-lg"}>{t("content.validityTimesOptions.thrice")}</label>
+                <label data-testid={"DataShareContent-Validity-Times-DropDown-NoLimit"} onClick={()=>{dispatch(storeVCExpiryTimes(-1)); setTimesDropDown(false)} } className={"w-full h-full py-3 px-4 hover:bg-iw-borderLight hover:rounded-lg"}>{t("content.validityTimesOptions.noLimit")}</label>
+                <label data-testid={"DataShareContent-Validity-Times-DropDown-Custom"} onClick={()=>{setTimesDropDown(false); props.setCustom(true)} }
+                       className={"w-full h-full py-3 px-4 hover:bg-iw-borderLight hover:rounded-lg"}>{t("content.validityTimesOptions.custom")}</label>
+            </div>
+        </div>}
+        <DataShareDisclaimer content={t("disclaimer")}/>
+    </div>;
+}
+
+export type DSContentType = {
+    credentialName: string;
+    credentialLogo: string;
+    setCustom: (custom: boolean) => void;
```

Comment:
what is this `custom`?

Reply:
Renamed it to "IsCustomExpiryInTimesModalOpen"